lib/README.md

```diff
@@ -22,6 +22,11 @@ rely on a file compiled in this directory.
 Contains code for extracting kernel data - i.e. all input and output parameters of
 a kernel invocation.
 
+## `nan_test`
+
+Contains PSyData libraries for checking that input and output parameters of kernels
+are valid number (i.e. not NAN or infinite).
```

Comment:
"are valid numbers". Also "NaN" and "infinity" (or just "Inf").

Reply:
Done.